pkg/profiler/cpu/cpu.go

```diff
@@ -396,12 +430,19 @@ type (
 	// https://dave.cheney.net/2015/10/09/padding-is-hard
 	// TODO(https://github.com/parca-dev/parca-agent/issues/207)
 	stackCountKey struct {
-		PID           uint32
+		PID           int32
 		UserStackID   int32
 		KernelStackID int32
+		_             int32
```

Comment:
I assume this is a padding, because of embedding another struck or because of the layout of the embedded struct. It would be nice to add a comment about it.

Reply:
Good shout!! Yes, it's padding as we have 3 int32 before 64-bit aligned fields